src/js/common/schemaform/review/ReviewCollapsibleChapter.jsx

```diff
@@ -93,6 +93,9 @@ export default class ReviewCollapsibleChapter extends React.Component {
     const hasUnViewedPages = this.pageKeys.some(key => !viewedPages.has(key));
 
     const ChapterDescription = chapter.reviewDescription;
+    let chapterTitle = chapter.title;
+    if (typeof chapter.title === 'function') chapterTitle = chapter.title(true);
+    if (chapter.reviewTitle) chapterTitle = chapter.reviewTitle;
```

Comment:
@jbalboni is this what you had in mind? that title could be fed an argument that indicated whether it was being used on a review page?

Reply:
Yep, that'll work, though I think having single line `if`s without braces is confusing to read.

Reply:
Good point, updated.